api/swagger_v2.yaml

```diff
@@ -0,0 +1,1138 @@
+            payload.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+        '401':
+          description: >-
+            Unauthorized.  RBAC prevented operation from executing, or
+            authentication token has expired.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+        '404':
+          description: Does Not Exist.  Endpoint not available.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+        '405':
+          description: >-
+            Operation Not Permitted.  For /scn, only POST operations are
+            allowed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+        '500':
+          description: >-
+            Internal Server Error.  Unexpected condition encountered when
+            processing the request.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem7807'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/StateChanges'
+        required: true
+components:
+  requestBodies:
+    SubscribePost:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/SubscribePost'
+      required: true
+    SubscribePostV2:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/SubscribePostV2'
+      required: true
+  schemas:
+    SubscribePost:
+      title: State Change Notification Subscription Message Payload
+      type: object
+      description: >-
+        This is the JSON payload that contains State Change Notification
+        subscription information.
+      properties:
+        Components:
```

Comment:
In the example you have:

```{
  "Components": [
    "x0c1s2b0n3"
  ],
  "Enabled": true,
  "Roles": [
    "Compute"
  ],
  "SoftwareStatus": [
    "AdminDown"
  ],
  "States": [
    "Ready"
  ],
  "Url": "https://x0c1s2b0n3.cray.com:8080/scns"
}
```

This is a bit confusing b/c this would say that x0c1s2b0n3 is subscribing to itself?  I dont think thats the intent?  Can we tweak the examples?

Reply:
Fixed